Source: Exherbo, Quentin “Sardem FF7” Glidic <sardemff7@exherbo.org>
Upstream: no
Reason: 1) use PKG_CONFIG 2) fontconfig is underlinked through xft, and gold does not allow that
    (can't use PKG_PROG_PKG_CONFIG easily, because aclocal.m4 is no generated by aclocal)

diff -ruN tk8.5.18.orig/unix/configure.in tk8.5.18/unix/configure.in
--- tk8.5.18.orig/unix/configure.in	2015-03-06 15:14:59.000000000 +0000
+++ tk8.5.18/unix/configure.in	2015-07-14 12:24:56.789555801 +0100
@@ -463,6 +463,10 @@
 #	Check for freetype / fontconfig / Xft support.
 #--------------------------------------------------------------------
 
+if test $PKG_CONFIG = "" ; then
+    PKG_CONFIG="pkg-config"
+fi
+
 if test $tk_aqua = no; then
     AC_MSG_CHECKING([whether to use xft])
     AC_ARG_ENABLE(xft,
@@ -475,15 +479,10 @@
 	AC_MSG_RESULT([$enable_xft])
     else
 	found_xft="yes"
-	dnl make sure package configurator (xft-config or pkg-config
-	dnl says that xft is present.
-	XFT_CFLAGS=`xft-config --cflags 2>/dev/null` || found_xft="no"
-	XFT_LIBS=`xft-config --libs 2>/dev/null` || found_xft="no"
-	if test "$found_xft" = "no" ; then
-	    found_xft=yes
-	    XFT_CFLAGS=`pkg-config --cflags xft 2>/dev/null` || found_xft="no"
-	    XFT_LIBS=`pkg-config --libs xft 2>/dev/null` || found_xft="no"
-	fi
+	dnl make sure package configurator (pkg-config)
+	dnl says that xft and fontconfig are present.
+	XFT_CFLAGS=`${PKG_CONFIG} --cflags xft fontconfig 2>/dev/null` || found_xft="no"
+	XFT_LIBS=`${PKG_CONFIG} --libs xft fontconfig 2>/dev/null` || found_xft="no"
 	AC_MSG_RESULT([$found_xft])
 	dnl make sure that compiling against Xft header file doesn't bomb
 	if test "$found_xft" = "yes" ; then
